src/Application/Console/Commands/InvokableCommand.php

```diff
@@ -34,6 +34,9 @@ public function __construct(string $name, callable $callable, InputDefinition $d
 
     public function execute(InputInterface $input, OutputInterface $output): int
     {
-        return call_user_func($this->callable, $input, $output);
+        $result = call_user_func($this->callable, $input, $output);
+
+        $output->writeln('✨ See something that needs to be improved? <options=bold>Create an issue</> or send us a <options=bold>pull request</>: <fg=cyan;options=bold>https://github.com/nunomaduro/phpinsights</>');
```

Comment:
Make sure this line is not getting piped to a file now when using json formatter. 

Because I think it will be now.